lib/users/lms/award-badge.js

```diff
@@ -14,10 +15,14 @@ function awardLMSBadge (args, cb) {
   var user = certif.user;
 
   function checkTestStatus (waterfallCb) {
-    if (certif.header.webHookType !== 'course_completion')
-      return cb(null, 'Unhandled webhook');
-    if (certif.enrollmentStatus !== 'passed' && certif.enrollmentStatus !== 'completed' )
-      return cb(null, 'Unhandled status');
+    if (certif.header.webHookType !== 'course_completion') {
+      return cb(null, {ok: false, why: 'Unhandled webhook'});
+    }
+    if (!_.every(certif.modules, {'status': 'passed'}) &&
+      !_.every(certif.modules, {'status': 'completed'}) &&
+      !_.includes(['completed', 'passed'], certif.enrollmentStatus) ) {
```

Comment:
That was my thought abt the fact that OR would be more accurate, as the !every are a fallback for includes, which is supposed to work